src/client/browser/idle-page/index.html.mustache

```diff
@@ -2,13 +2,30 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <title></title>
-    <script type="application/x-javascript" src="/browser/assets/index.js"></script>
-    <script type="text/javascript">
-        init('{{{statusUrl}}}');
-    </script>
+    <script type="text/javascript" src="/browser/assets/index.js"></script>
+    <link rel="stylesheet" href="/browser/assets/styles.css"/>
+    <link rel="icon" type="image/x-icon" href="/favicon.png"/>
 </head>
 <body>
-{{{userAgent}}}
+<div class="page-background">
+    <div class="container">
+        <canvas class="spinner" width="400px" height="400px"></canvas>
```

Comment:
We have a fixed size here, but does it looks ok on mobile devices? Shouldn't we scale size according to the viewport size?